phpBB/install_new/startup.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+/** @ignore */
+if (!defined('IN_PHPBB') || !defined('IN_INSTALL'))
+{
+	exit;
+}
+
+function phpbb_require_updated($path, $optional = false)
+{
+	global $phpbb_root_path, $table_prefix;
```

Comment:
That's true, though it could be injected into the container latter, but we don't have the class_loader yet, that was it i think. :)